requirements.txt

```diff
@@ -1,3 +1,4 @@
 Jinja2==2.10
 lxml==4.3.2
 requests==2.21.0
+certifi
```

Comment:
You don't think we should pin?

Reply:
Seems like you basically always want the latest version -- there's no real API, so no significant concerns for backwards compatibility?

Reply:
Okay, I suppose it just sets off my reproducible build instincts.

OTOH, this is a project that does nothing by query the outside world.